GithubReleaseScript/github_release.py

```diff
@@ -0,0 +1,231 @@
+"""
+This script can be used to build the message for the release in GitHub. It will fetch commits between
+two versions, make the diff and properly tag users with @mention. Contributors will be listed in the message,
+ignoring users emails that are listed in the 'author_exception_list'.
+
+The script can either be used with manual versions input, or the script can fetch latest released versions
+from the App Store.
+
+To use you first need to install python requirements through pip in your virtual environment.
+Then create a `config.py` file that will contain the username and token to access GitHub API with format:
+username = "username"
+token = "123456"
+
+Two versions can be passed in as arguments:
+- python script.py -o 'oldVersion' -n 'newVersion'
+- python script.py -o 103.1 -n 104.0
+
+Or no versions can be passed in, and the script will retrieve the versions from the App Store:
+- python script.py
+"""
+
+import argparse
+import config
+import git
+import requests
+import re
+import os
+import time
+
+from dataclasses import dataclass
+from github import Github
+from typing import Optional
+
+
+@dataclass
+class Version:
+    major: int
+    minor: int
+
+    @property
+    def string_name(self):
+        return f"{self.major}.{self.minor}"
+
+    @classmethod
+    def build_version(cls, version_string: str):
+        if "." in version_string:
+            split = version_string.split(".")
+            return cls(int(split[0]), int(split[1]))
+        else:
+            return cls(int(version_string), 0)
+
+
+# The Firefox for iOS App Store URL
+firefox_url = 'https://apps.apple.com/ca/app/firefox-private-safe-browser/id989804926'
```

Comment:
Is it safe to commit this id? I guess if you have added it but just to double check :)